program/write-a-program-to-check-anagram-string/WriteAProgramToCheckAnagramString.java

```diff
@@ -0,0 +1,33 @@
+import java.util.*;
+
+
+class WriteAProgramToCheckAnagramString{
+	public static void main(String[] args){
+		// Inputs
+		String s1 = "pan";
+		String s2 = "nap";
+
+		//Check anagrams
+		if(isAnagram(s1,s2)){
+			System.out.println("Anagram Strings");
+		}else{
+			System.out.println("Not Anagram Strings");
+
+		}
+
+
+}
+	public static boolean isAnagram(String str1, String str2) {
+		// Convert strings to character arrays
+		char[] arr1 = str1.toCharArray();
+		char[] arr2 = str2.toCharArray();
+
```

Comment:
It should accept case sensitive strings as well .
Can you convert the given string to lowercase and then split to an array, so that it will work irrespective of the case.

Reply:
Yeah sure, that will be more effective.

Reply:
I have made the requested changes. Please look into that.